records_mover/records/types.py

```diff
@@ -58,6 +58,8 @@ class UrlDetailsEntry(TypedDict):
                                           total=False)
 
     RecordsFormatType = Literal['delimited', 'parquet']
+
+    DelimitedVariant = Literal['dumb', 'csv', 'bigquery', 'bluelabs', 'vertica']
```

Comment:
There's room to expand use of this into records mover proper, but it'll require a bit of code moving to do the validation up front - maybe a follow-up PR.